site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -182,6 +183,45 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `agones.controller.mutatingWebhook.annotations`          | [Annotations][annotations] added to the Agones mutating webhook                                                                                                                                                                                                                                                  | `{}`                                      |
 | `agones.controller.mutatingWebhook.disableCaBundle`      | Disable ca-bundle so it can be injected by cert-manager                                                                                                                                                                                                                                                          | `false`                                   |
 | `agones.controller.allocationBatchWaitTime`              | Wait time between each allocation batch when performing allocations in controller mode                                                                                                                                                                                                                           | `500ms`                                   |
+{{% /feature %}}
+
+{{% feature publishVersion="1.37.0" %}}
```

Comment:
Out of curiosity - what's the difference between this table and the above one?

Reply:
She took out the references to the feature gate, e.g. `agones.controller.replicas` previously had: `(Ignored unless [SplitControllerAndExtensions][split-controller] is enabled)`

Reply:
Ah thanks! I didn't see it 👍🏻

Reply:
We have removed '(Ignored unless [SplitControllerAndExtensions][split-controller] is enabled)' from the description of a few parameters.